crates/matrix-sdk-ui/src/room_list_service/state.rs

```diff
@@ -126,6 +132,42 @@ impl Action for AddVisibleRoomsList {
     }
 }
 
+struct SetVisibleRoomsListToZeroTimelineLimit;
```

Comment:
Overall, with respect to naming those actions, I think we could get rid of `Rooms` and `Lists` in those, since that's what we're manipulating everywhere with these actions, so they wouldn't be such a mouthful :-)